.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/cuda/gups/hammerblade-helpers"]
+	path = examples/cuda/gups/hammerblade-helpers
+	url = git@github.com:mrutt92/hammerblade-helpers
```

Comment:
I feel like this should be somewhere else in the directory tree long-term, but since you're the only one using it right now, LGTM

Reply:
Yeah there's a PR that was never merged.

Reply:
See #724

Reply:
Wondered. Do you want that merged, or keep this as a submodule for now?

Reply:
I don't know. We should talk about it.

Reply:
Urgently?

Reply:
nah, this is something we can fix later.

Reply:
I'm more concerned with getting this merged so it doesn't go stale.